### PR TITLE
feat(watch): apply when file changes (mk2)

### PR DESCRIPTION
**Pull request by malcolmholmes**
Tweaks to #13 - cool stuff.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -9,4 +9,5 @@ require (
 	github.com/grafana-tools/sdk v0.0.0-20200326200416-f0431e44c1c3
 	github.com/kylelemons/godebug v1.1.0
 	github.com/spf13/pflag v1.0.5
+	gopkg.in/fsnotify.v1 v1.4.7
```
Comment (Duologic) [inline go.mod:12]: Maybe remove the one from github...

File: pkg/dash/workflow.go [hunk 2]
```diff
@@ -74,29 +75,29 @@ func Diff(config Config, jsonnetFile string, targets *[]string) error {
...
+			fmt.Println(difference)
 		}
 	}
 	return nil
 }
 
 // Apply renders a Jsonnet dashboard then pushes it to Grafana via the API
 func Apply(config Config, jsonnetFile string, targets *[]string) error {
-	folderId, err := folderId(config, jsonnetFile)
+	folderID, err := folderId(config, jsonnetFile)
```
Comment (Duologic) [inline pkg/dash/workflow.go:89]: Unrelated change, but ok.

Comment (malcolmholmes) [reply pkg/dash/workflow.go:89]: editor asks for it!

File: go.mod (hunk 1 above)
Comment (malcolmholmes) [reply go.mod:12]: okay

**Pull request merged**